maps_dashboards/public/utils/geo_point_formater.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const geoJSONTypes = [
+  'point',
+  'linestring',
+  'polygon',
+  'multipoint',
+  'multilinestring',
+  'multipolygon',
+  'geometrycollection',
+];
+
+export function isGeoJSON(value: any) {
```

Comment:
How about change the value type to `value: unknown` or to a more specific type, like `value: object`?

Reply:
updated